### PR TITLE
Updates to about pages & footer to highlight board search/hiring

### PR DESCRIPTION
**Pull request by clottman**
## Links
* no remix - pull this branch and test locally if you're interested
* https://app.clubhouse.io/glitch/story/3041/update-site-to-promote-hiring-board-search
* See also [doc with more details on the requirements](https://docs.google.com/document/d/1eQKaFNxbx9qnPMy1vbtlH1HsEgLt2OKz5GjfrCTQfAc/edit?usp=sharing)

## GIF/Screenshots:
careers page

company page

footer

## Changes:
* Add a section highlighting the board search to the careers page beneath the lever section, and the company page after advisors
* Add a link to the careers page from the footer, under 'About Glitch'

## How To Test:
* Pull this branch locally, build and view about/careers and about/company. 

## Feedback I'm looking for:
Any glaring styling issues here?

### CODE REVIEW

File: src/components/about/index.js [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+import classnames from 'classnames';
 import styles from './about.styl';
+import pageStyles from '../../presenters/pages/about/about.styl';
 
-export const BlockSection = ({ children, className }) => <section className={classNames(styles.blockSection, className)}>{children}</section>;
+export const BlockSection = ({ children, className }) => <section className={classnames(styles.blockSection, className)}>{children}</section>;
+
+export const BoardSection = ({ markColor, useDefaultHeadingSize }) => (
+  <section className={classnames(pageStyles.backgroundSection, styles.boardSection)}>
+    <Heading tagName="h2" className={classnames({ [pageStyles.h2]: !useDefaultHeadingSize }, styles.boardHeading)}>
```
Comment (ttseng) [inline src/components/about/index.js]: This heading should live outside the section for consistency

image

File: src/components/about/about.styl [hunk 2]
```diff
@@ -47,3 +47,10 @@
 .boardMember
   font-size: 70%
   font-weight: bold
+
+.boardHeading 
+  padding-top: 10px
+
+.boardSection
```
Comment (ttseng) [inline src/components/about/about.styl]: if we push the heading outside of `.boardSection` then the padding should be `padding: 5px 12px`

File: src/components/footer/index.js [hunk 3]
```diff
@@ -31,6 +31,10 @@ const Footer = ({ containerClass }) => (
       About Glitch
       <Icon className={emoji} icon="crystalBall" />
     </FooterLink>
+    <FooterLink href="/about/careers" track="careers">
+      We're Hiring
+      <Icon className={emoji} icon="fishingPole" />
```
Comment (ttseng) [inline src/components/footer/index.js]: love this choice!

File: src/components/about/index.js (hunk 1 above)
Comment (clottman) [reply src/components/about/index.js]: Better now? https://new-about-pages.glitch.me/about/careers
image

**Pull request merged**